test/powershell/Modules/Microsoft.WSMan.Management/CredSSP.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+Describe "CredSSP cmdlet tests" -Tags 'Feature','RequireAdminOnWindows' {

```

Comment:
Actually it appears you have explicitly run `start-pspester` with `-unelevated`.  I'll have to take care of this myself in the script then.